### ISSUE TITLE
Sort does not work as expected

### ISSUE THREAD

**Issue by zchsh** [marmelab/aor-graphql#42]
Wanted to try sorting a Resource by a specific field, and the sort order always seemed off - correct within the page of results, but not correct across multiple pages.

Let's say I have 30 results, displaying 10 per page, and I sort them ascending, by their IDs. I'd expect the following:
Page 1: IDs 0-9
Page 2: IDs 10-19
Page 3: IDs 20-29
Instead, I see the following:
Page 1: IDs 20-29
Page 2: IDs 10-19
Page 3: IDs 0-9

In `buildVariables.js` around line 80, the parameters `skip`, `last` and `orderBy` seem relevant. I logged them out, and here's how they relate to the above weirdly ordered pages:
Page 1: IDs 20-29 - skip: 0, last: 10
Page 2: IDs 10-19 - skip: 10, last: 10
Page 3: IDs 0-9 - skip: 20, last: 10

It seems the effect these parameters seem to have is:
`orderBy` - first the elements are ordered as requested, in this case, 0-29
`skip` - next, the first `skip` # of elements are removed. if `last` is set, they seem to be removed from the end of the ordered elements, if `first` is set, they're removed from the start.
`last` - finally, the last `last` # of elements are returned.

In short, in the return of the function `buildGetListVariables` in `buildVariables.js`, by just changing
```js
last: parseInt(params.pagination.perPage),
```
to
```js
first: parseInt(params.pagination.perPage),
```
the problem seems to be solved.

Let me know if that makes any sense! I can't seem to find any documentation on `skip`...

**Comment by djhi** [comment]
Fixed by #48
